README.md

```diff
@@ -64,40 +66,43 @@ To get started with developing the frontend, you will need NodeJS & Python 3.
 
 Set up the Python portion of the program by running:
 
-```
+```bash
 $ virtualenv venv
 $ source venv/bin/activate
 $ pip install -e .
 ```
 
-Then to download the cached parts list and process it, run:
+Then to download the cached parts list (from the GH Pages site, generated via GH Actions) and process it, run:
```

Comment:
Again, I find this extra information rather confusing - like I can see it is downloaded from the GH pages from the URL. And GH pages are in 99+% generated in GH actions.

Reply:
GH Pages are not in 99% of GH Actions. GH Actions generally run unit tests. I know that you think it's obvious. I'm an experienced dev, and I did not think it's obvious, hence why I spent my weekend updating these docs. If you think I'm stupid for not knowing, feel free to request that I revert this change.

Reply:
You misinterpreted my saying (and inverted the implication):
- I say "something is provided via GH Pages" implies "it was probably built using CI, most probably GH Actions"
- You read "We use GH Actions" implies "We have GH Pages"

What I dislike is "random pieces of information" scattered across the documentation. If you want to improve something, there should be "architecture overview" that captures the high-level design. It shouldn't pollute step-by-step instruction that the user is expected to just follow.

Reply:
As I said in another thread - a new document describing the overall architecture. But building such a document is currently out of my time scope.

Reply:
If it's out of your timescope, can you just accept this change, and I'll split it out into a separate file later?